src/STAT.md

```diff
@@ -42,34 +42,34 @@ Mode 0  ___000___000___000___000___000___000________________000
 Mode 1  ____________________________________11111111111111_____
 ```
 
-When the PPU is reading a particular part of video memory,
-that memory is inaccessible to the CPU.
+When the PPU is accessing some video-related memory, that memory is inaccessible
+to the CPU: writes are ignored, and reads return garbage values (usually $FF).
 
-- During modes 2 and 3, the CPU cannot access OAM (FE00h-FE9Fh).
-- During mode 3, the CPU cannot access VRAM or CGB Palette Data
-  (FF69,FF6B).
+- During modes 2 and 3, the CPU cannot access [OAM](<#VRAM Sprite Attribute Table (OAM)>) ($FE00-FE9F).
+- During mode 3, the CPU cannot access VRAM or [CGB palette data registers](<#FF69 - BCPD/BGPD (Background Color Palette Data or Background Palette Data) - CGB Mode Only>)
+  ($FF69,$FF6B).
 
-Mode | Action                                                           | Duration                                                           | Accessible video memory
+Mode | Action                                                      | Duration                                                           | Accessible video memory
 -----|------------------------------------------------------------------|--------------------------------------------------------------------|-------------------------
-  2  | Searching OAM for OBJs whose (X,Y) coordinates overlap this line | 80 dots (19 µs)                                                    | VRAM, CGB palettes
-  3  | Reading OAM and VRAM to generate the picture                     | 168 to 291 dots (40 to 60 µs) depending on sprite count            | None
-  0  | Horizontal blanking                                              | 85 to 208 dots (20 to 49 µs) depending on previous mode 3 duration | VRAM, OAM, CGB palettes
-  1  | Vertical blanking                                                | 4560 dots (1087 µs, 10 scanlines)                                  | VRAM, OAM, CGB palettes
+  2  | Searching OAM for OBJs whose Y coordinate overlap this line | 80 dots (19 µs)                                                    | VRAM, CGB palettes
+  3  | Reading OAM and VRAM to generate the picture                | 168 to 291 dots (40 to 60 µs) depending on sprite count            | None
+  0  | Nothing (horizontal blanking)                               | 85 to 208 dots (20 to 49 µs) depending on previous mode 3 duration | VRAM, OAM, CGB palettes
```

Comment:
```suggestion
  0  | Nothing (HBlank)                               | 85 to 208 dots (20 to 49 µs) depending on previous mode 3 duration | VRAM, OAM, CGB palettes
```

Reply:
I figured spelling out the name in full wouldn't hurt, since we have some room to spare.

Reply:
I don't think it's a question of space here, as in the entire document (after the definition we use HBlank and VBlank), one can think we are talking about something different if go back at this spelling in some tables..

Reply:
Fair enough. (And one more reason to #194, lol.)